tests/phpunit/HttpExtractorTest.php

```diff
@@ -23,54 +25,88 @@ class HttpExtractorTest extends TestCase
     /** @var mixed[] */
     private $history = [];
 
+    /** @var TestHandler */
+    private $testHandler;
+
+    /**
+     * @return mixed[][]
+     */
+    public function provideUrlsAndExceptions(): array
+    {
+        return [
+            'HTTP 404' => [
+                array_fill(0, 5, new Response(404, [], '')),
+                UserException::class,
+                'Server returned HTTP 404 for "http://example.com"',
+            ],
+            'HTTP 401' => [
+                array_fill(0, 5, new Response(401, [], '')),
+                UserException::class,
+                'Server returned HTTP 401 for "http://example.com"',
+            ],
+            'HTTP 500' => [
+                array_fill(0, 10, new Response(500, [], '')),
+                UserException::class,
+                'Server returned HTTP 500 for "http://example.com"',
+            ],
+            'HTTP 503' => [
+                array_fill(0, 10, new Response(503, [], '')),
+                UserException::class,
+                'Server returned HTTP 503 for "http://example.com"',
+            ],
+        ];
```

Comment:
to je pekne blbej napad menit neco v kodu v testech a jeste k tomu prerovnavat metody

Reply:
To teda jo. To byl nějakej flake ve formátování asi...

Reply:
Fixed

Reply:
Měl jsem z nějakých pokusů omylem zapnuté "reorganize code" u formátování.